Hush/Assets/Scripts/Weapon/Spell.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Enums;
+
+public class Spell : MonoBehaviour, IWeapon
+{
+    [SerializeField] private Animator animator;
+    public int CurrentDamage { get; set; }
+    public int BonusDamage { get; set; }
+
+    private const int SPECIAL_DAMAGE = 50;
+
+    void Awake()
+    {
+        BonusDamage = 5;
```

Comment:
### Minor tweak
Could be better to separate the property & its getter/setter that way you can auto assign it a value. by default BUT also have the value from the inspector get assigned should you decide to overwrite it in the inspector. 
ie:
``` C#
[SerializeField] private int bonusDamage = 5; // actual property
public int BonusDamage 
{
    get => bonusDamage; 
    set => bonusDamage = value;
 }
```

Reply:
sounds good!